### ISSUE TITLE
w12extra: wrong URL to course site

### ISSUE THREAD

**Issue by sadphi** [lunduniversity/introprog#621]
The URL in w12 exercise 3a is incorrect, which means that the wrong data is fetched. The URL in the compendium is [https://cs.lth.se/pgk/lthkurser](https://cs.lth.se/pgk/lthkurser), and when running the code in the REPL instructions, `data` will be equal to 

```scala
val data: String =
<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>nginx/1.20.1</center>
</body>
</html>
```

This is due to the URL redirecting to [https://fileadmin.cs.lth.se/pgk/lthkurser201819.txt](https://fileadmin.cs.lth.se/pgk/lthkurser201819.txt). Replacing the URL with the latter gives the expected result.

**Comment by bjornregnell** [comment]
PR welcome to avoid redirect. (I'm not sure how to do `fromURL` to bypass the redirect)

**Comment by sadphi** [comment]
Created PR #624 with a fix

**Comment by sadphi** [comment]
This was partly solved in PR #624. However, it will probably break when a new snapshot is released. Closing the issue though since it most likely requires a fix in the backend.
